src/test/java/org/terracotta/statistics/simulation/latency/Percentile.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * All content copyright Terracotta, Inc., unless otherwise indicated.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics.simulation.latency;
+
+import org.apache.commons.math3.stat.descriptive.DescriptiveStatistics;
+import org.terracotta.statistics.derived.latency.LatencyHistogramStatistic;
+
+import java.text.NumberFormat;
+import java.util.function.DoubleUnaryOperator;
+
+/**
+ * @author Mathieu Carbou
+ */
+public class Percentile {
+
+  private final double value;
+  private final double min;
+  private final double max;
+
+  private Percentile(double value, double min, double max) {
+    this.value = value;
+    this.min = min;
+    this.max = max;
+  }
+
+  public double value() {
+    return value;
+  }
+
+  public double min() {
+    return min;
+  }
+
+  public double max() {
+    return max;
+  }
+
+  public double[] bounds() {
+    return new double[]{min, max};
+  }
+
+  @Override
+  public String toString() {
+    return min == max ? String.valueOf(value) : (value + " [" + min + ", " + max + "]");
```

Comment:
Can't this call the other toString with a formatter?

Reply:
Oh! yes I can now remove the `toString()`! I was using it before for JFreeChart (to display chart title), but not anymore